frontend/pages/DetailedViewPage.js

```diff
@@ -26,11 +26,31 @@ export default class DetailedViewPage extends Component {
         this.setState(({priceOverlayVisible}) => ({
             priceOverlayVisible: !priceOverlayVisible
         }));
-    }
-
-    changePrice = () => {
+    };
 
-    }
+    changePrice = async () => {
+        let storeName = this.props.navigation.state.params.store;
+        let lat=0, long=0;
+        for (let i in this.state.stores) {
```

Comment:
I see that you are searching for the first store that matches what was received as prop but you can probably explain that in a comment for readability 👍